frontend/src/components/review/CommentsPanel.tsx

```diff
@@ -34,27 +33,51 @@ const CommentsPanel = ({ storyTranslationId }: CommentPanelProps) => {
     setFilterIndex(event.target.selectedIndex);
   };
 
-  const commentsComponents = comments.map((comment: CommentResponse) => (
-    <Text key={comment.id}>{JSON.stringify(comment)}</Text>
-  ));
+  const CommentsComponent = () => {
```

Comment:
`CommentsSidebar`? `CommentsPanel`?

Reply:
`CommentsList` is more literal to what this component is. `CommentsSidebar` and `CommentsPanel` dont work since `CommentsPanel` is already defined.
I'm fine with CommentsComponent for now since it's going to get a little weird once both standard comments and WIP comments are happening, along with threads. 
I'll leave it up to lynn!

Reply:
I also thought the naming was a bit odd, changing it to CommentsList for now